renovate.json

```diff
@@ -1,13 +1,7 @@
 {
-  "extends": [
-    "config:base"
-  ],
-  "ignorePaths": [
-    "test/"
-  ],
-  "labels": [
-    "Type: Dependencies"
-  ],
+  "extends": ["config:base"],
```

Comment:
I don't understand the logic that prettier is using here. This is the complete opposite of the css issue.

Reply:
Formatting tries to be consistent across languages when possible, but some languages are very different and can't have 1:1 matching, like CSS vs. <anything>.

In general, they try to fit on one line if it's not too long (for arrays, because for objects that doesn't apply if you've intentionally broken it up into multiple lines), but with CSS, it's always 1 statement per line.